src/main/java/ch/njol/skript/events/EvtCommand.java

```diff
@@ -30,54 +32,60 @@
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
-@SuppressWarnings("unchecked")
-public class EvtCommand extends SkriptEvent { // TODO condition to check whether a given command exists, & a conditon to check whether it's a custom skript command
+public class EvtCommand extends SkriptEvent {
 	static {
-		Skript.registerEvent("Command", EvtCommand.class, CollectionUtils.array(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class), "command [%-string%]")
-				.description("Called when a player enters a command (not necessarily a Skript command) but you can check if command is a skript command, see <a href='conditions.html#CondIsSkriptCommand'>Is a Skript command condition</a>.")
-				.examples("on command:", "on command \"/stop\":", "on command \"pm Njol \":")
-				.since("2.0");
+		Skript.registerEvent("Command", EvtCommand.class, CollectionUtils.array(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class), "[1:existing] command [using] [%-strings%]")
```

Comment:
`Existing` sounds bad, you should find a different word like `real` maybe.

Reply:
could be something like `registered`